deploy/template.yaml

```diff
@@ -123,6 +126,8 @@ objects:
             value: http
           - name: ASSISTED_SERVICE_HOST
             value: "assisted-service:8090"
+          - name: IMAGE_SERVICE_HOST
```

Comment:
that's the var we use in the code: https://github.com/openshift/assisted-image-service/blob/f6af8ed8e6e6c1f6df0ba4a0301c488d620ab007/main.go#L39

Reply:
yeah I was wondering why we need to have a difference between those two names (because I do like better the external name)
but if that might be a problem for backwards-compatibility then no need to change it